common/scala/src/main/scala/org/apache/openwhisk/core/WhiskConfig.scala

```diff
@@ -186,56 +186,56 @@ object WhiskConfig {
 }
 
 object ConfigKeys {
-  val cluster = "whisk.cluster"
-  val loadbalancer = "whisk.loadbalancer"
-  val buildInformation = "whisk.info"
+  val cluster = "org.apache.openwhisk.cluster"
```

Comment:
I do not think we need to change the configKeys?

Reply:
We need to change everything based on the Apache policy.

Reply:
I'm not convinced we need the extra levels here.  It should be further discussed.

Even if we eventually decide to make the change, it absolutely should be a separate PR.  Changing the name of config variables is significantly higher impact on downstream projects than changing the names of the Scala/Java packages